man/coredumpctl.xml

```diff
@@ -217,6 +218,13 @@
         will be used. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--debugger-args=</option><replaceable>ARGS</replaceable></term>
```

Comment:
not a fan of the abbreviation "args". Please call this --debugger-arguments=. This is a long option anyway, no need to abbreviate it needlessly.

That said, maybe add a short switch for this too? `-A` or so?